src/components/Icons/_Icons.scss

```diff
@@ -47,6 +47,10 @@
         height: 18px;
         width: 18px;
     }
+
+    &--negative {
```

Comment:
Should this be coded into the styling of the icon itself, or should it be already implemented on the client side and the only thing that's using it is Storybook?